### ISSUE TITLE
Add a copy of our licence to the the website (Trac #881)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#967]
At present our licence information only appears on !GitHub which is not where users (as opposed to developers) will look for it. Add the license text to www.sasview.org as well.

Migrated from http://trac.sasview.org/ticket/881
```json
{
    "status": "closed",
    "changetime": "2017-03-11T16:06:50",
    "_ts": "2017-03-11 16:06:50.517490+00:00",
    "description": "At present our licence information only appears on !GitHub which is not where users (as opposed to developers) will look for it. Add the license text to www.sasview.org as well.",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Admin",
    "time": "2017-03-10T11:25:54",
    "component": "SasView",
    "summary": "Add a copy of our licence to the the website",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.1.0",
    "owner": "smk78",
    "type": "task"
}
```

**Comment by butlerpd** [comment]
Trac update at `2017/03/11 16:03:04`: **butler** changed description from "At present our licence information only appears on GitHub which is not where users (as opposed to developers) will look for it. Add the license text to www.sasview.org as well." to "At present our licence information only appears on !GitHub which is not where users (as opposed to developers) will look for it. Add the license text to www.sasview.org as well."

**Comment by butlerpd** [comment]
Trac update at `2017/03/11 16:06:50`: 
* **butler** commented:

Steve King added the information and committed with the "closes #967" tag.  Unfortunately the website repo (io) is not tied to the trac system.  Closing ticket manually now.


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "new" to "closed"
